README.md

```diff
@@ -343,31 +343,6 @@ All dbt commands are supported.
 
 ### Custom configurations
 
-#### General
-
-* *Enable view column types in docs* -  Teradata Vantage has a dbscontrol configuration flag called `DisableQVCI` (QVCI - Queryable View Column Index). This flag instructs the database to build `DBC.ColumnsJQV` with view column type definitions. 
-
-    > :information_source: Existing customers, please see [KB0022230](https://support.teradata.com/knowledge?id=kb_article_view&sys_kb_id=d066248b1b0000187361c8415b4bcb48) for more information about enabling QVCI.
-
-  To enable this functionality you need to:
-  1. Enable QVCI mode in Vantage. Use `dbscontrol` utility and then restart Teradata. Run these commands as a privileged user on a Teradata node:
-      ```bash
-      # option 551 is DisableQVCI. Setting it to false enables QVCI.
-      dbscontrol << EOF
-      M internal 551=false
-      W
-      EOF
-
-      # restart Teradata
-      tpareset -y Enable QVCI
-      ```
-  2. Instruct `dbt` to use `QVCI` mode. Include the following variable in your `dbt_project.yml`:
-      ```yaml
-      vars:
-        use_qvci: true
-      ```
-      For example configuration, see `test/catalog/with_qvci/dbt_project.yml`.

```

Comment:
There is a section in this file at the end regarding Unit Testing, which talks about QVCI
Can we please remove that section as well ?!